packages/tokamak/contracts/deploy-config/holesky-nightly.ts

```diff
@@ -0,0 +1,20 @@
+const config = {
+  numDeployConfirmations: 1,
+  l1BlockTimeSeconds: 15,
+  l2BlockGasLimit: 15_000_000,
+  l2ChainId: 4906,
+  ctcL2GasDiscountDivisor: 32,
+  ctcEnqueueGasCost: 60_000,
+  sccFaultProofWindowSeconds: 10,
+  sccSequencerPublishWindowSeconds: 12592000,
+  ovmSequencerAddress: '0x7DF510Ec916D8099c8A6C9979878246Fd94247Aa',
```

Comment:
This address does not have any assets. (https://holesky.etherscan.io/address/0x7DF510Ec916D8099c8A6C9979878246Fd94247Aa)
Is it a valid address?

Reply:
Yes, I didn't send to each address yet. I will send it before creating network!

Reply:
Please leave information of accounts to https://onther.atlassian.net/wiki/spaces/OPTIMISTIC/pages/78413838/Accounts

Reply:
I fixed it. Thank you!

Reply:
I updated it! Thank you!